### ISSUE TITLE
Recreating a deleted project cannot add admin using new-project

### ISSUE THREAD

**Issue by liggitt** [openshift/origin#1083]
Deleting a project leaves its resources and policy in place (which is a different issue)

Recreating a deleted project fails assigning admin because the 'admin' rolebinding already exists

```
project=ui-test-project7
openshift ex new-project $project --admin="anypassword:createuser"
openshift ex policy add-user admin anypassword:adduser -n $project
osc describe policybinding master -n $project
osc delete project $project
openshift ex new-project $project --admin="anypassword:createuser2"
openshift ex policy add-user admin anypassword:adduser2 -n $project
osc describe policybinding master -n $project
```

As a user, I'd expect `new-project --admin ...` to be equivalent to `new-project`+`add-user` calls, but `add-user` is better at adding the user to the role, regardless of what role bindings are in place.

**Comment by liggitt** [comment]
@deads2k

**Comment by liggitt** [comment]
Fixed via #1173
